caracal/workers/crosscal_worker.py

```diff
@@ -128,7 +129,18 @@ def do_KGBF(i):
         gtable_ = None
         ftable_ = None
         interp = RULES[term]["interp"]
-        params["refant"] = pipeline.refant[iobs] or '0'
+        if pipeline.refant[iobs] in ['auto']:
+            params["refant"] = manants.get_refant(pipeline, recipe,
+                                                  prefix, msname, fields,
+                                                  pipeline.minbase[iobs],
+                                                  pipeline.maxdist[iobs], i)
+            if params["refant"]:
+                caracal.log.info(f"Auto selected ref antenna(s): {params['refant']}")
+            else:
+                caracal.log.error("Cannot auto-select ref antenna(s). Set it manually.")
```

Comment:
This line  should cause CARACal to exit.

Currently it does not, the pipeline continues with  `refant=''` and crashes.

Reply:
I meant L.140 of course.

Reply:
And yes, I agree that `refant: auto` should  be the default.